webapp/src/main/webapp/groovy/news/webui/activity/UINewsStreamActivity.gtmpl

```diff
@@ -224,7 +224,7 @@
 			</div><!--end heading-->
 			<div class="description <%=isArchived ? "newsArchived" : ""%>">
           <img src="$illustrationURL" class="newsSmallIllustration" alt="News"/>
-          <div class="newsDetails">
+          <div class="newsDetailsApp">
```

Comment:
need to be reverted to newsDetails to keep css

Reply:
ok, IMO, it's better to add the class `newsDetails` in https://github.com/exoplatform/news/blob/develop/webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue#L2 instead of here to centralize the definition of parent class in Vue component that will be used in Full Vue application

Reply:
This class is needed also here for stream activities not for full activities (news detail)